guides/common/modules/ref_system-requirements.adoc

```diff
@@ -41,6 +41,10 @@ endif::[]
 * A system umask of 0022
 * Full forward and reverse DNS resolution using a fully-qualified domain name
 
+NOTE: The shortnames in the hostnames are no longer supported for the {Project} and {SmartProxy} server.
+Also, the custom certificate's Common Name (CN) must have a fully qualified domain name (FQDN) instead of shortnames.
+This does NOT apply to the clients of a {Project} and they can continue to use shortnames if required by the user.
```

Comment:
Never use caplocks. 
We don't use an article before {Project}
```suggestion
This does not apply to the clients of {Project}.
```
Minimalism.

Reply:
We talk to the user, not about the user.

Reply:
Noted, Melanie. Taking changes 👍

Reply:
Modified.